File: FitnessApp/app/src/main/java/com/example/fitnessapp/data/model/Exercise.kt
package com.example.fitnessapp.data.model

data class Exercise(
    val exerciseId: Int,
    val name: String,
    val muscleGroup: String?,
    val description: String?,
    val equipment: String?,
    val difficulty: String?,
    val previewImage: String?,
    val tutorialImage: String?,
)

File: FitnessApi/FitnessApi/Models/Exercise.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace FitnessApi.Models;

public partial class Exercise
{
    public int ExerciseId { get; set; }

    public string Name { get; set; } = null!;

    public string? MuscleGroup { get; set; }

    public string? Description { get; set; }

    public string? Difficulty { get; set; }

    public string? Equipment { get; set; }

    public string? PreviewImage { get; set; }

    public string? TutorialImage { get; set; }
    [JsonIgnore]
    public virtual ICollection<WorkoutCompletionExercise> WorkoutCompletionExercises { get; set; } = new List<WorkoutCompletionExercise>();
    [JsonIgnore]
    public virtual ICollection<WorkoutExercise> WorkoutExercises { get; set; } = new List<WorkoutExercise>();
}
